### PR TITLE
Basic Digitalpour parser

### PR DESCRIPTION
**Pull request by mjcarroll**

### CODE REVIEW

File: tap_list_providers/parsers/digitalpour.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+if __name__ == '__main__':
+    import argparse
+
+    locations = {
+        'sta': ('5761f0a45e002c13703ed811', 1),
+        'wywbmad': ('57b130dd5e002c0388f8b686', 1),
+        'wywb805': ('57b130dd5e002c0388f8b686', 2),
+        'otbx': ('5502506cb3b70304a8f2e0d2', 1),
+        'rccb': ('5aa1a8135e002c0924805971', 1),
+        'bufeddies': ('5afe0f3a5e002c0b8060a5b8', 1)
```
Comment (drewbrew) [inline tap_list_providers/parsers/digitalpour.py]: I don't like putting their API keys into code, even for example purposes.

Comment (mjcarroll) [reply tap_list_providers/parsers/digitalpour.py]: These aren't API keys, these are the companyIDs, retrieved from the API endpoint: https://mobile.digitalpour.com/DashboardServer/v4/MobileApp/StoreLocations?ApiKey=574725e55e002c0b7cf0cf19

File: tap_list_providers/parsers/digitalpour.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+import json
+
+import datetime
+import dateutil.parser
+import requests
+
+class DigitalPourParser:
+    """Parser for DigitalPour based vendors."""
+
+    URL = 'https://mobile.digitalpour.com/DashboardServer/v4/MobileApp/MenuItems/{}/{}/Tap?ApiKey={}'
```
Comment (drewbrew) [inline tap_list_providers/parsers/digitalpour.py]: put a `  # noqa` at the end of this line to silence the line length warning (two spaces before `#` required)

Comment (drewbrew) [reply tap_list_providers/parsers/digitalpour.py]: ... or not, apparently that isn't too long

File: tap_list_providers/parsers/digitalpour.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        for price in prices:
+            p = {
+                'size': price['Size'],
+                'price': price['Price'],
+                'per_ounce': price['Price']/price['Size']
+            }
+            pricing.append(p)
+        return pricing
+
+
```
Comment (drewbrew) [inline tap_list_providers/parsers/digitalpour.py]: delete this extra blank line

**Pull request merged**